.gitignore

```diff
@@ -5,6 +5,7 @@
 *.so
 *.dylib
 ./tape
+tape
```

Comment:
no need?

Reply:
em... or this is just t avoid tape binary here?

Reply:
after I execute `make tape` there is a binary there so I am just excluding this binary for uploading

Reply:
I see.